datasets/utils/continual_dataset.py

```diff
@@ -10,6 +10,11 @@
 import torch.nn as nn
 import torch.optim
 from torch.utils.data import DataLoader, Dataset
+import torchvision.transforms as transforms
+from datasets.transforms.driftTransforms import DefocusBlur, GaussianNoise, ShotNoise, SpeckleNoise
```

Comment:
transforms and driftTransforms are unused in this file

Reply:
Right, removed those import.